contributors/devel/pull-requests.md

```diff
@@ -77,35 +106,240 @@ $ $RELNOTES -man # for details on how to use the tool
 $ $RELNOTES --branch=master
 ```
 
-## Visual overview
+Release notes apply to PRs on the master branch. For cherry-pick PRs, see the [cherry-pick instructions](cherry-picks.md). The only exception to these rules is when a PR is not a cherry-pick and is targeted directly to the non-master branch.  In this case, a `release-note-*` label is required for that non-master PR.
+
+Now that your release notes are in shape, let's look at how the PR gets tested and merged.
+
+## The Testing and Merge Workflow
+
+The Kubernetes merge workflow uses comments to run tests and labels for merging PRs. NOTE: For pull requests that are in progress but not ready for review, prefix the PR title with "WIP" and track any remaining TODOs in a checklist in the pull request description.
+
+Here's the process the PR goes through on its way from submission to merging.
+
+1. Make the pull request
+1. mergebot assigns a reviewer
+
+If you're **not** in the whitelist:
+
+1. Reviewer suggests edits
+1. Push edits to your PR branch
+1. Repeat the prior two steps as needed
+1. Once all comments have been addressed, squash the commits
+1. Reviewer adds the labels `lgtm` and `ok-to-merge` which triggers the final tests and merge process (this can be done with the command `/lgtm`), or, if they are a whitelisted contributor, can add the comment `@k8s-bot ok to test` which triggers the intermediate tests
```

Comment:
I'm not sure `ok-to-merge` is still used. I think once review is done, and the reviewer feels that it's safe to test, then they go `@k8s-bot ok to test`. also should call out approval process, which is separate from review